src/org/labkey/test/tests/StudySurveyTest.java

```diff
@@ -0,0 +1,139 @@
+package org.labkey.test.tests;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.categories.DailyB;
+import org.labkey.test.pages.EditDatasetDefinitionPage;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.PortalHelper;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.List;
+
+@Category({DailyB.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 10)
```

Comment:
This test took 1m24s on [TeamCity](https://teamcity.labkey.org/viewLog.html?order=DURATION_DESC&recordsPerPage=20&filterText=&status=&currentGroup=test&scope=org.labkey.test.Runner%23teamcity%23junit.framework.TestSuite.org.labkey.test%23teamcity%23tests&pager.currentPage=1&buildTypeId=bt7&buildId=965199&tab=testsInfo). Reduce timeout so that its a bit more representative of actual runtime.
```suggestion
@BaseWebDriverTest.ClassTimeout(minutes = 3)
```

Reply:
I was going to change the time, after the teamcity run.

Thank you